src/angular-google-analytics.js

```diff
@@ -187,13 +192,29 @@
         };
 
         var _ga = function() {
+          var tmpCommand, tmpArguments;
           if (!$window.ga) {
             that._log('warn', 'ga function not set on window');
           }
           if (logAllCalls === true) {
             that._log.apply(that, arguments);
           }
-          $window.ga.apply(null, arguments);
+          // If multiple trackers && arguments valid && command supports multiple trackers
+          // Temporary check if command contains '.' until all functions are switched
+          if (angular.isArray(accountId) && arguments && arguments.length > 1 && arguments[0].indexOf('.') < 0 &&
+              trackingSupportingMultipleTrackers.indexOf(arguments[0]) >= 0) {
+            // Create command for each tracker an apply
+            tmpArguments = arguments
+            accountId.forEach(function (trackerObj) {
+              tmpCommand = tmpArguments[0];
+              tmpCommand = trackerObj.name + '.' +tmpCommand;
+              tmpArguments[0] = tmpCommand;
+              $window.ga.apply(null, tmpArguments);
```

Comment:
This implementation is going to have problems due to tmpArguments being set to arguments on line 207. On line 211 the 0 index then gets overwritten and that will be used on the following iteration. The tests should have caught this not working correctly.

The following will work correctly and removes the need for line 195 and 207 as well.
```
var command = trackerObj.name + '.' + arguments[0],
      args = arguments.slice();
args[0] = command;
$window.ga.apply(null, args);
```